ansible/.gitignore

```diff
@@ -1,2 +1,3 @@
 inventory/inventory.yml
 ansible.cfg
+roles/ssh-initial/vars/main.exmaple.yml
```

Comment:
Why the example? Wouldn't be better to exclude `main.yml` with personal data?

Reply:
the problem is that the format is not clear if you leave the file out

Reply:
uh sorry you are correct